### DISCUSSION TITLE
Issues with 1.7.0

### DISCUSSION THREAD

**Discussion by emilinsweden** [w35l3y/hassio-addons#23] [Q&A] [answered]
Hello the add on is going offline with the new update.
![image](https://user-images.githubusercontent.com/11667571/230134738-e0e635a0-84f5-46e2-9711-ce096bbf7c65.jpeg)

**Comment by w35l3y** [comment]
Share your settings and ommit sensitive data, please.

It failed while adding routes.
Probably, one of your services aren't ok.
Were you using this add-on before version 1.7.0 ?

**Reply by emilinsweden** [reply]
Hi,yes we have talked before. I have had it for about a year I think. Now that it got updates it suddenly stopped running so I have to manually start it, but then it stops after a while. I don't know what I need info is sensitive so please let me know! Thanks

![image](https://user-images.githubusercontent.com/11667571/230160957-55d9bab5-ae49-4fc2-b356-c846a3a98b35.png)

![image](https://user-images.githubusercontent.com/11667571/230160976-f923d3ba-565a-4ab6-bb84-e286c27b6183.png)

![image](https://user-images.githubusercontent.com/11667571/230160997-c8ffe9a4-a675-483f-b945-1023a602628a.png)

![image](https://user-images.githubusercontent.com/11667571/230161023-9edf6097-2398-409c-998f-e1dc771b0dbb.png)

**Reply by emilinsweden** [reply]
![image](https://user-images.githubusercontent.com/11667571/230161384-57ebe207-b9cb-46fc-b97e-9d7a3f96675e.png)

**Reply by emilinsweden** [reply]
I don't know why it stopped running. I restarted home assistant now but it keeps running

**Reply by w35l3y** [reply, marked as answer]
![Screenshot_20230405_153218_Samsung Internet](https://user-images.githubusercontent.com/963510/230172953-e647159b-0eee-49b0-b598-965a2741007d.jpg)

Change metrics from `localhost:41705` to `0.0.0.0:41705`


![image](https://user-images.githubusercontent.com/963510/230174487-4da11c9e-6195-44b4-825a-5468e266c370.png)

**Reply by emilinsweden** [reply]
I Will try it thank you. I really appreciate your help :)!

**Reply by w35l3y** [reply]
Have you tested it already ?

**Reply by emilinsweden** [reply]
Yes! Still going strong. I'll let you know in case it goes down again. Thanks

**Reply by emilinsweden** [reply]
Hi now my other Home assistant instance is halting the add-on. I have two on one cloud flare account.
![image](https://user-images.githubusercontent.com/11667571/233403367-46a909eb-fe6a-4ccb-b248-9ec2ac34f4ec.png)

**Reply by w35l3y** [reply]
https://github.com/w35l3y/hassio-addons/tree/main/cloudflare_tunnel#common-errors

> Watchdog missing application response from c50d1fa4_cloudflare_tunnel
Change metrics to `0.0.0.0:41705` in the config.
For details, check https://github.com/w35l3y/hassio-addons/issues/22#issuecomment-1491623708
